lib/octokit/client/apps.rb

```diff
@@ -33,10 +33,12 @@ def find_integration_installations(options = {})
       #
       # @see https://developer.github.com/v3/apps/#list-installations-for-user
       #
-      # @return [Array<Sawyer::Resource>] A list of installations
+      # @return [Sawyer::Resource] A list of installations
```

Comment:
I believe this returns a `Sawyer::Resource` containing a `total_count` and installations`, which is an array.